v19.1/rename-database.md

```diff
@@ -25,6 +28,27 @@ Parameter | Description
 ----------|------------
 `name` | The first instance of `name` is the current name of the database. The second instance is the new name for the database. The new name [must be unique](#rename-fails-new-name-already-in-use) and follow these [identifier rules](keywords-and-identifiers.html#identifiers). You cannot rename a database if it is set as the [current database](sql-name-resolution.html#current-database) or if [`sql_safe_updates = true`](set-vars.html).
 
+## Database renaming considerations
```

Comment:
Both database and table renaming is not transactional. Is there already something written up for table renaming?

Reply:
I don't think so.  There is nothing on the [`RENAME TABLE`](https://www.cockroachlabs.com/docs/dev/rename-table.html) page about that.

On the known limitations page under [Schema changes within transactions](https://www.cockroachlabs.com/docs/v19.1/known-limitations.html#schema-changes-within-transactions), we do say that

> Within a single transaction:
> ...
> A table cannot be dropped and then recreated with the same name. This is not possible within a single transaction because DROP TABLE does not immediately drop the name of the table. As a workaround, split the DROP TABLE and CREATE TABLE statements into separate transactions.

which seems related but is not explicitly talking about table renames.

Is the mechanism for table renaming basically the same as for database renaming?  If so I can:

1. update the RENAME TABLE docs with this information as well
2. update the known limitations page to say that database _and_ table renames are not transactional

Reply:
yes please update those as well and make this a "Database and Table renames are not transactional" update

Thanks!

Reply:
Done.  PTAL.  And thanks for the review!